modules/xmpp/strophe.jingle.js

```diff
@@ -376,10 +376,13 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
                 }
 
                 if (options.useStunTurn) {
-                    // we want to filter and leave only tcp/turns candidates
-                    // which make sense for the jvb connections
-                    this.jvbIceConfig.iceServers
-                        = iceservers.filter(s => s.urls.startsWith('turns'));
+                    // By default we filter out STUN and TURN/UDP and leave only TURN/TCP.
+                    let filter = s => s.startsWith('turn') && s.urls.indexOf('transport=tcp');
```

Comment:
Optional: Not a big deal but I would do the actual assignment in `else` to avoid defining a function that is not used.